examples/client.c

```diff
@@ -1092,6 +1108,31 @@ cmdline_option(char *arg) {
     coap_insert_optlist(&optlist,
                         coap_new_optlist(num, strlen(arg), (unsigned char *)arg));
   }
+  if (num == COAP_OPTION_PROXY_SCHEME) {
+    proxy_scheme_option = 1;
+    if (strcmp(arg, "coaps+tcp") == 0) {
```

Comment:
These should be `strcasecomp()`. (Also in the server).

Reply:
Code updated and pushed.